sdk-actors/src/main/java/io/dapr/actors/client/ActorProxyBuilder.java

```diff
@@ -132,7 +105,7 @@ public T build(ActorId actorId) {
             this.actorType,
             actorId,
             this.objectSerializer,
-            buildDaprClient());
+            this.channel.getDaprClient());
```

Comment:
If the channel just calls `getDaprClient()` is there a ready why it needs to be a separate class and not **just** a dapr client?

Reply:
At this point, it makes sense to make it simply implement the DaprClient interface and forward the calls to the actual client implementation underneath. It reads better and gives sense of simplicity to users: `new ActorClient()`. I also renamed it to `ActorClient` as I think it is the best name for this class right now. `DaprClient` (No `I` in Java's naming convention here) is the interface and I did not want to use `DaprClientImpl`.